tools/ratbagctl.body.py.in

```diff
@@ -1429,9 +1429,9 @@ class RatbagParserSwitch(RatbagParser):
 
     def _print_help(self, prefix: str) -> None:
         if self.help and self.group is not None:
-            string = self.help.split("\n")
-            string = "\n  ".join(string)
-            print(" ", string, "\n")
+            help_lines = self.help.split("\n")
+            help_indented = "\n  ".join(help_lines)
+            print(" ", help_indented, "\n")
```

Comment:
/me stare at this

I think we can use `print(textwrap.indent(self.help, '  '))` here instead of of the `split` + `join`.

Reply:
This is much better, thanks!